lib/mocha/expectation.rb

```diff
@@ -292,6 +294,13 @@ def with_no_block_given
     #   sum # => 1
     #   fibonacci.next_pair { |first, second| sum = first + second }
     #   sum # => 2
+    #
+    # @example Yield the provided block
```

Comment:
Similar to above, this would be more accurate:

```suggestion
    # @example Yield to the block provided when calling
```

But then, how is this different from the [first example](https://github.com/freerange/mocha/pull/496/files#diff-4c63dbab29ecbfb14e097947e3e0b295ad1382843978026979062d85b5287288R274-R280)? This example might be simpler than the other one, being focused only on yielding, rather than yielding followed by returning as in the first example. So, we can consider replacing the first one with this example. In essence, though, they appear to be the same to me.

Am I missing something?

Reply:
@nitishr I agree with your suggested change to the wording - thanks. As I mentioned elsewhere, I'm happy to add more examples or simplify/split the existing ones if it makes things clearer.